HouseholdManager/Views/Room/Details.cshtml

```diff
@@ -1,38 +1,46 @@
-﻿@model HouseholdManager.Models.Room
+﻿@model HouseholdManager.Models.ViewModels.RoomDetailViewModel
 @using HouseholdManager.Data.API
 
 @{
-    ViewData["Title"] = "Room Details";
+    ViewData["Title"] =  @Model.RoomName + " Details";
     var req = new IconRequestor();
-    string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
-                                                                 (IEnumerable<Icon>)ViewBag.Icons);
+    //Below is no longer needed with the ViewModel in place
+    //string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
+    //                                                             (IEnumerable<Icon>)ViewBag.Icons);
```

Comment:
Hey, so, this call actually still doing something. We only store the icon emoji grapheme in the database (as a string) and not anything else about it, so this is taking that from the ViewModel to then get the mouseover text from the API.

I know, we have both an "Icon" class and an "Icon" property with different data types, and that's something I'd definitely want to rework if given the time.

```suggestion
    string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,(IEnumerable<Icon>)ViewBag.Icons);
```